plugins/flowdock/app/assets/javascripts/flowdock/jquery_mentions_input/jquery.mentionsInput.js

```diff
@@ -0,0 +1,564 @@
+/*
```

Comment:
Can't do it at the moment. I had to fork the jquery-mentions-input to add support for a default value on the textarea. Submitted the [PR#131](https://github.com/podio/jquery-mentions-input/pull/131). When its accepted then we can switch.